cultivate/theme.json

```diff
@@ -0,0 +1,339 @@
+{
+	"customTemplates": [
+		{
+			"name": "blank",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Blank"
+		},
+		{
+			"name": "header-footer-only",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Header and Footer Only"
+		},
+		{
+			"name": "footer-only",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Footer Only"
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"custom": true,
+			"customGradient": true,
+			"palette": [
+				{
+					"color": "#1E5044",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#FFDB61",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#000000",
+					"name": "Foreground",
+					"slug": "foreground"
+				},
+				{
+					"color": "#FAF5EF",
+					"name": "Background",
+					"slug": "background"
+				},
+				{
+					"color": "#729B79",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			]
+		},
+		"custom": {
+			"gap": {
+				"horizontal": "min(40px, 5vw)",
+				"vertical": "min(40px, 5vw)"
+			}
+		},
+		"layout": {
+			"contentSize": "650px",
+			"wideSize": "1210px"
+		},
+		"spacing": {
+			"units": [
+				"%",
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
+			]
+		},
+		"typography": {
+			"customFontSize": true,
+			"fontFamilies": [
+				{
+					"fontFamily": "Chivo, sans-serif",
+					"slug": "body-font",
+					"name": "Body (Chivo)",
+					"google": "family=Chivo:ital,wght@0,400;0,700;1,400;1,700",
+					"fontSlug": "chivo"
+				},
+				{
+					"fontFamily": "Rubik, sans-serif",
+					"slug": "heading-font",
+					"name": "Headings (Rubik)",
+					"google": "family=Rubik:ital,wght@0,400;0,700;1,400;1,700",
+					"fontSlug": "rubik"
+				}
```

Comment:
These fonts are both rendering as the browser default for sans-serif rather than the custom font:

<img width="1381" alt="image" src="https://user-images.githubusercontent.com/1645628/171062272-3d957a42-aa41-4557-b7a9-f9a767d4f655.png">

This is probably because this style of font setting only works in Blockbase. Outside of Blockbase, you should set custom fonts [like this](https://github.com/Automattic/themes/blob/trunk/curator/theme.json#L93):

```
{
	"fontFamily": "'Work Sans', sans-serif",
	"name": "Headings (System Font)",
	"slug": "heading-font",
	"fontFace": [
		{
			"fontDisplay": "block",
			"fontFamily": "'Work Sans'",
			"fontWeight": "900",
			"fontStyle": "normal",
			"fontStretch": "normal",
			"src": [
				"file:./assets/fonts/WorkSans.ttf"
			]
		}
	]
}
```

You'll also need to bundle the font files [with the theme](https://github.com/Automattic/themes/tree/trunk/curator/assets/fonts).

Reply:
Alright, I bundled the fonts with the theme and updated theme.json.

Isn't it a little weird though that we need to include fonts this way even though they are available on WordPress.com? AFAIK this theme will not be released on .org.

Reply:
> Isn't it a little weird though that we need to include fonts this way even though they are available on WordPress.com?

Oh yeah you're right! Maybe the fonts need to be done slightly differently for WordPress.com.. I guess they're handled by Jetpack?

I've just tested setting Chivo and Rubik as font families in theme.json on .com, without bundling the font files themselves, and it seems to work fine. It looks like you should be able to use something like the following:

```
	"fontFamilies": [
		{
			"fontFamily": "\"Chivo\", sans-serif",
			"slug": "chivo"
		},
		{
			"fontFamily": "\"Rubik\", sans-serif",
			"slug": "rubik"
		}
	]
```

And then there's no reason to include the font files either. Sorry! 🙈 This is much simpler.